docs/content/commands/npm-deprecate.md

```diff
@@ -10,6 +10,8 @@ description: Deprecate a version of a package
 npm deprecate <pkg>[@<version range>] <message>
 ```
 
+Note: This command is unaware of workspaces.
```

Comment:
this definitely should be workspace-aware, since folks will want to deprecate individual (or multiple) workspaces.

Reply:
ok yeah, this comment is a clue on how we can improve the message... the true purpose of this note is to let users aware that there's no point trying to run this command with a `--workspaces` or `--workspace=<name>` configs 🤔

Reply:
of course folks will be able to deprecate a package even if it's manages as a workspace somewhere

Reply:
but i mean, i might want to deprecate N packages at a time, based on workspace config.

Reply:
def a valid usecase 🤔 honestly I think the confusion factor here is due to the current syntax of `npm deprecate` that doesn't read any info from the current dir `package.json` file, it only accepts a single <pkg>@<version> argument along with a message, which makes it harder to scale to multiple pkgs or workspaces

Reply:
that said, it sounds more like something we're going to have to redesign later and probably in a breaking change way, so I guess for now it will still have to be marked here as a command that is unaware of the workspaces configs.

Reply:
opened a RFC to follow through: https://github.com/npm/rfcs/issues/341

thanks @ljharb 👍